.github/workflows/build_and_test.yml

```diff
@@ -433,11 +407,32 @@ jobs:
         key: scala-213-coursier-${{ hashFiles('**/pom.xml', '**/plugins.sbt') }}
         restore-keys: |
           scala-213-coursier-
-    - name: Install Java 11
+    - name: Install Java 8
       uses: actions/setup-java@v1
       with:
-        java-version: 11
+        java-version: 8
     - name: Build with SBT
       run: |
         ./dev/change-scala-version.sh 2.13
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Djava.version=11 -Pscala-2.13 compile test:compile
+
```

Comment:
@dongjoon-hyun, I just move it back below. Seems like it affects when things are ported back (Hadoop 2 build is only in master branch). I think it will still obviously show the red X one of builds fails anyway .. so I think it might be fine to move below here together with Java 11 and Scala 2.13 builds.

Reply:
@HyukjinKwon . This was added at the end but was moved to the first position due @mridulm 's direct request.

Reply:
@mridulm, would you mind moving it below here? It will show an X obviously if one of builds fails anyway. It seems like it can cause some conflicts easily when we backport something to other branches for GitHub Actions. Currently, we're running the builds in all active branches, and porting back things.

Reply:
@HyukjinKwon Currently, we merge if either of the builds pass (jenkins or github).
The reason I was requesting @dongjoon-hyun to move it to top was if jenkins gives positive build (since it is not testing hadoop 2), then only way to identify hadoop 2.x failure is by github actions failing on hadoop 2 build.

If we can get jenkins to also build hadoop-2, then we dont need to move this to top.
Else, it is common for reviewers to not see the last entry in github build (when there is a jenkins green).

Reply:
@HyukjinKwon Is there a better solution currently ? I am not an expert on github or jenkins :-)

Reply:
@mridulm, oh but if there's any of them fails, it shows a clear X like

![Screen Shot 2020-11-18 at 9 07 02 PM](https://user-images.githubusercontent.com/6477701/99528700-077e5680-29e2-11eb-9769-f27b414657da.png)

So it should be fine. I do believe reviewers checks the X or test failures from Jenkins before merging it in :-). Actually, Java 11 and Scala 2.13 doo. PR builder does not run them as well.

Reply:
That sounds good to me. Thanks for the details @HyukjinKwon !
@dongjoon-hyun given details @HyukjinKwon provided, I am fine with the changes he proposed. Thanks

Reply:
Thanks!

Reply:
Given the agreement, shall we merge this?

Reply:
Please go ahead @dongjoon-hyun , thanks !